### DISCUSSION TITLE
More submarines in the ocean

### DISCUSSION THREAD

**Discussion by Likvidus** [FakeFishGames/Barotrauma#6209] [Ideas and suggestions]
This is one of my old ideas. From time to time during the trip (even without a mission) we will be able to meet neutral submarines following their way to other stations or just wandering in the ocean, submarines of various factions (Coalition, separatists, etc. with all the ensuing consequences: a bad reputation will lead to a battle), unfriendly submarines (bandit, pirate, mercenary), mining shuttles, reconnaissance drones, underwater mines and so on. When another submarine approaches ours, they send a message (greeting, death threats, etc.) so that we can understand what we are dealing with. All these boats are not marked on the map (for the element of surprise).
There is also an idea to add a chance to meet another submarine at an outpost (like ours), with its own crew.
I think it would be nice to create more of the illusion of life in the depths of an alien ocean.

**Comment by AshEdwardsDesign** [comment]
+1 this could be a really nice addition to make the world feel more "alive".

Additional idea: How about a "trader" sub that can dock with yours (if you accept), containing an NPC that sells some rarer items.

**Comment by Misterzelix** [comment]
+1 exactly this, maybe you can take over another sub when You run out of fuel
